.github/workflows/lint.yml

```diff
@@ -2,7 +2,9 @@
 # request and push to main.
 name: lint
 on:
-  workflow_call:
+  pull_request_target:
+    types:
+      - opened
```

Comment:
I'm unsure if this change is actually going to fix the issue, since its not running in CI for some reason? @MSevey